Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,210 @@
+
+### Step 13
+
+Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Euler Rotation Mutator` component. On the `Mutate on Axis` property turn off the `X` and `Z`  checkboxes.
+
+> The reason we only want the Y axis turned on is because we only want to rotate around this axis to make our capsule rotate to a new facing direction in the scene.
+
+![Drag Capsule Into Euler Mutator](assets/images/DragCapsuleIntoEulerMutator.png)
+
+### Step 14
+
+Select the `Rotation` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Transformed` event parameter on the `Float To Vector3` component.
+
+Drag and drop the `Rotation` GameObject into the event listener box that appears on the `Transformed` event parameter on the `Float To Vector3` component that displays `None (Object)`.
+
+![Drag Rotation Into Event Listener](assets/images/DragRotationIntoEventListener.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `TransformEulerRotationMutator` -> `DoSetProperty` function (be sure to select `Dynamic Vector3 - DoSetProperty` for this example).
+
+![Set Transformed Function](assets/images/SetTransformedFunction.png)
+
+### Step 15
+
+Expand the `Tilia Input CombinedActions Unity` Package directory in the Unity Project window and select the `Packages -> Tilia Input CombinedActions Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.AxesToAngle` prefab into the Unity hierarchy window.
+
+![Import Axes To Angle](assets/images/ImportAxesToAngle.png)
+
+### Step 16
+
+On the `Input.CombinedActions.AxesToAngle` GameObject, Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Vertical Axis` property on the `Axis To Angle Action` component.
+
+![Drag And Drop Axes To Angle Into Horizontal Axis](assets/images/DragAndDropAxesToAngleIntoHorizontalAxis.png)
+
+Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Vertical` Axis` property on the `Axis Settings` component.
+
+![Drag And Drop Axes To Angle Into Vertical Axis](assets/images/DragAndDropAxesToAngleIntoVerticalAxis.png)
+
+### Step 17
+
+Select the `Input.CombinedActions.AxesToAngle` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Value Changed` event parameter on the `Axes To Angle Action` component.
+
+Drag and drop the `Rotation` GameObject into the event listener box that appears on the `Value Changed` event parameter on the `Axes To Angle Action` component that displays `None (Object)`.
+
+![Create Axes To Angle Event](assets/images/CreateAxesToAngleEvent.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `Float To Vector3` -> `CurrentY` function (be sure to select `Dynamic Vector3 - CurrentY` for this example).
```

Comment:
Backtick is wrong here on `Float To Vector3` -> `CurrentY` should just be:

`Float To Vector3 -> CurrentY`